packages/story-editor/src/types/taxonomyProvider.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import type { APICallbacks } from './configProvider';
+
+export type Term = {
+  id: number;
+  link: string;
+  name: string;
+  slug: string;
+  taxonomy: string;
+  _links: Record<string, string>;
+};
+
+export type Taxonomy = {
+  restBase: 'web_story_category' | 'web_story_tag' | string;
```

Comment:
```suggestion
  restBase: string;
```

In order not to make any assumptions about WordPress context here